docker/docker-cluster/prometheus/prometheus.yml

```diff
@@ -26,3 +26,9 @@ scrape_configs:
     scrape_interval: 15s
     static_configs:
       - targets: ['localhost:9090']
+
+  - job_name: metrictank
+    metrics_path: /prometheus/metrics
+    scrape_interval: 10s
+    static_configs:
+      - targets: ['metrictank:6060']
```

Comment:
for the clustered ones, you'll probably want to scrape every metrictank instance 
in that case, i'm not sure if you can differentiate the different metrics (do they all get a unique host tag? or does it require separate scrape jobs? you'll need to try or ask a prometheus person)

Reply:
Sorry, I was careless there. I'll figure that out.

Reply:
Prometheus adds a label per host :)

![image](https://user-images.githubusercontent.com/11150914/56141393-05018280-5f9d-11e9-91d0-85076aac113a.png)

Reply:
Fixed in next commit